### DISCUSSION TITLE
Introduce second temp1 (3) instead of wait for copy

### DISCUSSION THREAD

**Discussion by aviator90** [madMAx43v3r/chia-plotter#721] [Ideas]
If waiting for copy I lose additional 10 Minutes per plot (copy is 5-600 seconds) if waiting before next plot starts, if normal ploting is done, then I get additional 3 minutes for the next plot to finish.
Perhaps introducing a second temp1 which is located on a different drive would be better, so copying is done from disk 1, but plotting is starting on disk 2 (temp2) and 3, so some kind of iteration like -G, but with 2 (3) different drives.

**Comment by Dartellum** [comment]
Leave off the -d and have a cron job or scheduled task to rsync/copy/move the file to the final destination in the background outside of `chia-plotter`. You would need enough space to do this though. Also, the copy from within `chia-plotter` works in the background and starts the next plot. If two plots are available is when you see the wait. See https://github.com/madMAx43v3r/chia-plotter/issues/662 and a few other issues.

**Reply by aviator90** [reply]
As stated the problem is not that the next plot would wait as in issue 662, it moves on to the next plot creation and the old is copied to the final directory, but it slows down the new plot (because it's reading from temp1 I think, and also writing to it) for around 3-5 mintues, which causes the new plot time to be higher..if I would wait for the copy to finish (parameter "-w") this would result in even higher numbers..8-10 minutes and that's why I suggested this feature.
That is why my suggestion is, to make an additional parameter for temp3 (serving as second temp1)...meaning
Plot is is on disk 1 (temp1, let's say C:/temp1), finished and being copied to H:/
Plot 2 is being created on disk 3 (E:/temp3) and is not interfering with the read process with writes
Temp2 is RamDisk or any other SSD disk (in my case I don't even use it, don't have enough Ram)
![image](https://user-images.githubusercontent.com/20906499/124400650-59cb7800-dd24-11eb-904d-91dded6a7c07.png)
